spotinst/ocean_ecs_launch_spec/fields_spotinst_ocean_ecs_launch_spec.go

```diff
@@ -921,6 +923,101 @@ func Setup(fieldsMap map[commons.FieldName]*commons.GenericField) {
 		nil,
 	)
 
+	fieldsMap[Strategy] = commons.NewGenericField(
+		commons.OceanECSLaunchSpec,
+		Strategy,
+		&schema.Schema{
+			Type:     schema.TypeSet,
```

Comment:
By convention, should be typeList.
See details about types here:
https://developer.hashicorp.com/terraform/plugin/sdkv2/schemas/schema-types

Reply:
implemented with reference of ocean_aws_launch_spec